client/jira.go

```diff
@@ -114,17 +114,16 @@ func GetIssueInfo(issueId string) (domain.Issue, error) {
 
 	statusCode, err := SendRequest(request, &issue)
 	if statusCode != http.StatusOK {
-		err = fmt.Errorf("issueId=%s : statusCode=%d", issueId, statusCode)
-		return domain.Issue{}, err
+		return domain.Issue{}, fmt.Errorf("issueId=%s : statusCode=%d", issueId, statusCode)
 	}
 	if err != nil {
-		err = fmt.Errorf("issueId=%s : err=%v", issueId, err)
```

Comment:
Since you removed this line `err = fmt.Errorf("issueId=%s : err=%v", issueId, err)` what will be the value for `err` in the next line?

Reply:
@YToutah993 good catch! I forgot to format the error when returning.

It should be
```go
if err != nil {
   return domain.Issue{}, fmt.Errorf("issueId=%s : err = %v", issueId, err)
}
```

Will update, thanks!